oop2-encapsulate-inherit.js

```diff
@@ -22,24 +122,135 @@
     
     - For each class, add 2 properties and 2 methods to each.  The properties and methods should make sense, considering what the classes are supposed to be representing.
 
+    //your code here...
```

Comment:
too much inheritance here...  A Site or Employee should not be an Umbrella.  Also, your constructor chaining is mismatched: you're calling super with parameter sets that don't have matching constructors

Reply:
reviewing the instructions, it's actually requesting this hierarchy, so just keep it in mind for the future (has-a vs is-a relationships)